index.md

```diff
@@ -174,8 +174,6 @@ Signed,
 
 [Andrey Unger](https://github.com/unger1984)
 
-[Denys Zelenov](https://github.com/torwalds)
-
```

Comment:
Please do not delete other signatures.

Reply:
I don't know how it happening( Maybe gh-editor bug

Reply:
Maybe it is merge conflict ?

Reply:
Because i see duplicates